rootfs/templates/wrapper-body.sh

```
@@ -0,0 +1,578 @@
+	echo "      terminal            Trace the terminal color mode detection"
+	echo "    You can specify multiple modes, separated by commas, e.g. --trace=custom,hist"
+	echo ""
+}
+
+function options_to_env() {
+	local kv
+	local k
+	local v
+
+	for option in "${options[@]}"; do
+		kv=(${option/=/ })
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)